api/models/spell.py

```diff
@@ -0,0 +1,229 @@
+"""
+The model for a spell.
+
+Also includes importing and some presentation logic.
+"""
+
+from django.db import models
+from django.template.defaultfilters import slugify
+from django.core.validators import MinValueValidator
+from django.core.validators import MaxValueValidator
+
+from .models import GameContent
+from ..validators import spell_school_validator
+
+
+class Spell(GameContent):
+    """The model for a spell."""
+
+    # Stored metadata about the spell
+    page = models.TextField(
+        help_text='Page number reference for the document.')
+
+    spell_level = models.IntegerField(
+        validators=[MinValueValidator(0), MaxValueValidator(9)],
+        help_text='Integer representing the level of the spell. Cantrip is 0.')
+
+    dnd_class = models.TextField(
+        help_text='List of classes (comma separated) that can learn this spell.')
+
+    school = models.TextField(
+        validators = [spell_school_validator],
+        help_text='Representation of the school of magic, such as "illusion" or "evocation".')
```

Comment:
As I described more thoroughly in `monster.py`, I'm interested in refactoring this a TextChoices field.

Reply:
Yes, this will be incoming relatively soon.